Tests/Unit/Fixtures/Hooks/TestingHookImplementor2.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types = 1);
+
+namespace OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks;
+
+use OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks\TestingHookInterface;
+
+/**
+ * Second test interface implementation to use with the HookProvider
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+class TestingHookImplementor2 implements TestingHookInterface
+{
+    /**
+     * @var bool
+     */
+    public static $wasCalled = 0;
+
+    public function __construct()
+    {
+        self::$wasCalled = 0;
+    }
+
+    /**
+     * This function will be called during HookProvider tests.
+     *
+     * @return void
+     */
+    public function testHookMethod()
+    {
+        self::$wasCalled++;
+    }
```

Comment:
For brevity, can this class extend the other so we don't have the code duplication?
(We probably then need a change from `self::$wasCalled` to `static::$wasCalled`.)